services/discovery/v1-query-builder.js

```diff
@@ -37,6 +37,20 @@ module.exports = function(RED) {
     res.json(serviceutils.checkServiceBound(SERVICE_IDENTIFIER));
   });
 
+  function processResponse(response, field) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 4 spaces but found 2.](https://app.codacy.com/manual/BetaWorks-NodeRED-Watson/node-red-node-watson/pullRequest?prid=4846228)